### PR TITLE
refactor spec tests, fix missing template issues

### PR DESCRIPTION
**Pull request by leoroos**
Following rubys template format is compatible with pjax and allows a consistent browser history behavior.
No missing templates, working back, forward, refresh and direct urls.

### CODE REVIEW

File: app/controllers/organizations_controller.rb [hunk 1]
```diff
@@ -16,7 +17,6 @@ def change_status
   end
 
   def sidebar
-    @organization_groups = Organization.all.group_by(&:status)
```
Comment (beckysiegel) [inline app/controllers/organizations_controller.rb:19]: Is sidebar necessary anymore? Looks empty

File: app/controllers/api_controller.rb [hunk 2]
```diff
@@ -20,6 +20,7 @@ def respond_to_format(response_data, filename)
         send_data export_organizations_as_docx(response_data), filename: "#{filename}.docx" 
       end
       format.markdown {render text: export_organizations_as_markdown(response_data)}
+      format.any {  raise ActionController::RoutingError.new('Not Found') }
```
Comment (c-lliope) [inline app/controllers/api_controller.rb:23]: It looks like there are two spaces before `raise`, which is probably why this was commented on.

File: app/assets/javascripts/organizations.js [hunk 3]
```diff
@@ -1,11 +1,17 @@
...
+$(document).pjax('a.edit-link-pjax', '#organization-content', {push: true});
+$(document).pjax('a.update-link-pjax', '#organization-content', {push: true});
 
 $(document).on('submit', '#edit-form form', function(event) {
   $.pjax.submit(event, '#organization-content', {push: false});
 });
 
+$(document).ready(function() {
+  if (('.org-address').length >= 1 ) {
+    displayMap($('.org-address').text());
```
Comment (c-lliope) [inline app/assets/javascripts/organizations.js:11]: We can ignore this for now, but in the future we should make sure we're requiring things correctly so we don't run into any dependency issues.

File: spec/controllers/api_controller_spec.rb [hunk 4]
```diff
@@ -17,4 +17,44 @@
...
+
+    it 'exports all docx' do
+      create(:organization, name: "organization docx")
+      get :export_all, format: 'docx'
+      #how to test for correctly exported docx?
+      expect(response.body).not_to be_nil
+    end
+
+    it 'requires format argument' do
+      expect {
```
Comment (c-lliope) [inline spec/controllers/api_controller_spec.rb:54]: this is fine, because we're chaining methods on the end of it.

**Comment by c-lliope** [comment]
@leoroos Looks good! - can you push a few more commits to fix the houndci comments?

Other than that, I think it's ready to merge!

**Comment by c-lliope** [comment]
Closing in favor of #32

**Pull request closed without merging**